### DISCUSSION TITLE
Fetching data

### DISCUSSION THREAD

**Discussion by riesi77** [mampfes/ha_epex_spot#137] [Q&A]
I just switched over to using the manual data fetching using the service call (epex_spot.fetch_data) at a certain time.
How can I turn off the automated fetching so it does not fetch multipe (senseless) times?
Thanks!

**Comment by mampfes** [comment]
There is no config option. You can remove the timer(s) that trigger the data fetch: https://github.com/mampfes/ha_epex_spot/blob/532eaec2e067621901807541e6bd8420862cd089/custom_components/epex_spot/__init__.py#L86

Just to be curios: Why do you fetch data manually?

**Reply by riesi77** [reply]
I extended my graph to show also the prices of tomorrow - if they are present already.  And for that I would like to fetch right after the time the providers update the data, for instance awattar at 14:00.
